### DISCUSSION TITLE
Jxrlib open pr or add to tap?

### DISCUSSION THREAD

**Discussion by Gcenx** [Homebrew/discussions#1367] [Everyday usage] [answered]
The current jxrlib formula produces .so files over the usual expected .dylib. I’ve taken the same approach as Debian and vcpkg but tweaked it to also provide a more correct .pc file.

I’ve modified the formula locally to produce .dylibs instead.

Side Note;\
If it’s best to keep this in my own tap should it be marked as conflicting with the jxrlib formula or would keg-only be the more brew correct way to handle this?

**Comment by carlocab** [comment, marked as answer]
If it's in your own tap it's really up to you whether you want to mark it as conflicting or as keg-only.

If you'd like to make the changes to the formula in homebrew/core, please open a pull request. It's hard to comment on changes without knowing exactly what they are (and without running them through CI).

**Reply by Gcenx** [reply]
Thanks for the response.

Still trying to get more familiar with how brew handles somethings as it’s vastly different than what I’m use to.
